prow/integ-suite-kind.sh

```diff
@@ -133,6 +133,7 @@ if [[ -z "${SKIP_SETUP:-}" ]]; then
     # Cluster 0, 1 and 4 share cluster 0's control plane
     # Clusters 2 and 3 control themselves
     export INTEGRATION_TEST_CONTROLPLANE_TOPOLOGY="0:0,1:0,2:2,3:3,4:0"
+    export INTEGRATION_TEST_CONFIG_TOPOLOGY="0:0,1:0,2:2,3:3,4:0"
```

Comment:
Is this saying that clusters 1 and 4 read config from cluster 1? The intention of the `INTEGRATION_TEST_CONTROLPLANE_TOPOLOGY` was to exercise all combinations of primary/remote and multi-network. We should have at most 1 cluster that uses a separate config cluster. 

We should either:
- add another cluster to our test topology (could be heavy)
- make a separate prow job for these topologies
- use cluster 0 for the control plane, cluster 1 as a config cluster, cluster 2 stays independent. Cluster 4 would then use cluster 3 as a control plane.

Reply:
I discussed with Iris yesterday ... the plan is:

- Switch multicluster to use the new istiod-less remote by default. We want to deprecate the legacy remote in 1.8.
- Create a prow job to test a single primary using a remote as a `config cluster`. So this job will probably just run some subset of the pilot tests to give us a warm fuzzy that config clusters actually work.

Reply:
SGTM